### PR TITLE
New App: CLT Lightrail

### PR DESCRIPTION
**Pull request by kconnell98**
Created an app that show real-time schedule for Charlotte Lightrail Trains (w/ All Stations)

### CODE REVIEW

**Comment by kconnell98** [comment]
recheck

**Comment by kconnell98** [comment]
recheck

**Comment by kconnell98** [comment]
Hey @betterengineering, @rohansingh, and @matslina, as a first time contributor, I was wondering if you can help explain why my build and test check failed? I was reading the logs and having a hard time figuring out what I am doing wrong.

Any help would be greatly appreciated!

**Comment by kconnell98** [comment]
I just tried one more thing to see if it will fix the build-and-test. In the meantime, I am attempting to attach a .gif file of the app. I followed your command (pixlet render apps/{{appname}}/{{app_name}}.star --gif --magnify 10) to create the file, but having a hard time pushing it. When I do 'git status' and 'git add .', it doesn't recognize the file and says it doesn't exist despite that not being the case. Am I missing something?

**Comment by rs7q5** [comment]
> I just tried one more thing to see if it will fix the build-and-test. In the meantime, I am attempting to attach a .gif file of the app. I followed your command (pixlet render apps/{{appname}}/{{app_name}}.star --gif --magnify 10) to create the file, but having a hard time pushing it. When I do 'git status' and 'git add .', it doesn't recognize the file and says it doesn't exist despite that not being the case. Am I missing something?

`.gif` is in the .gitignore. it's optional to add the gif anyways. So that's probably it. Uploading a gif would be old data at somepoint anyways so one could run locally. you could add the gif as a comment here at least to see it before it gets merged. I think you can force load a png too.  

TLDR. the gif is not absolutely necessary, but could drag and drop it in a comment here too to preview before it gets merged.  :D

**Comment by kconnell98** [comment]
Hey curious what the current state of this pull request is? The other day, all 3 checks were showing as approved. However, now the lint and build-and-test are showing that they are pending. Is this normal, or did I do something wrong on my end? Only thing I can recall was I tried merging the main branch into my repository after the checks for the pull request were approved. I assumed that I needed to have the most updated branch in my repository before I can start my next app, but would this mess with anything?

**Comment by rs7q5** [comment]
You can update the main branch without effecting this branch and make a new branch off of the updated main to work on a separate app, doing that should not disturb this branch that this PR is on, just make sure you have the right branch checked out locally. 

For reference:
managing branches: https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/making-changes-in-a-branch/managing-branches
about branches: https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/about-branches
